builder.config.json

```diff
@@ -191,16 +191,16 @@
         "is_interactive": true,
         "is_branching": false,
         "category":  1,
-        "menu_category": 4
+        "menu_category": 1,
```

Comment:
Curiosity Q: Are we using the `menu_category` to sort the block types between different categories in the new toolbar?

Reply:
I think no, this is just for menu.